connection.js

```diff
@@ -768,12 +806,20 @@ function customBoardsizeWidthsHeightsReject(argNameString, notificationWidth, no
         widthHeight = "height";
         notificationUnit = notificationHeight;
     }
-    conn_log(`boardsize ${widthHeight} ${rankedUnranked} -${notificationUnit}-, not in -${config[argNameString]}- `);
-    return { reject: true, msg: `In your selected board size ${notificationWidth} x ${notificationHeight} (width x height), boardsize ${widthHeight.toUpperCase()} (${notificationUnit}) is not allowed ${rankedUnranked} on this bot, please choose one of these allowed CUSTOM boardsize ${widthHeight.toUpperCase()}S values ${rankedUnranked}: ${config[argNameString]}` };
+    conn_log(`boardsize ${widthHeight} ${rankedUnranked} -${notificationUnit}-, `
+             + `not in -${config[argNameString]}- `);
+    const msg = `In your selected board size ${notificationWidth} `
+                + `x ${notificationHeight} (width x height), boardsize `
+                + `${widthHeight.toUpperCase()} (${notificationUnit}) `
+                + `is not allowed ${rankedUnranked} on this bot, please `
+                + `choose one of these allowed CUSTOM boardsize `
+                + `${widthHeight.toUpperCase()}S values ${rankedUnranked}: `
+                + `${config[argNameString]}`;
+    return { reject: true, msg };
 }
 
 function familyArrayFromFamilyNameString(familyNameString) {
-    return ["", "ranked", "unranked"].map(e => familyNameString + e);
+    return ["", "ranked", "unranked"].map(e => `${familyNameString}${e}`);
```

Comment:
Do we really need string interpolation here?

Reply:
i thought it was clearer than additions, should i remove them?

Reply:
mm yeah, actually this is ok.

Reply:
cool :)